test/install_script/nvm_install_with_aliased_dot

```diff
@@ -1,6 +1,8 @@
 #!/bin/sh
 
-setup () {
+setup() {
+  # Needed to avoid to checkout the repo to the latest nvm version, loosing the commits of the current PR
```

Comment:
```suggestion
  # Needed to avoid to checkout the repo to the latest nvm version, losing the commits of the current PR
```
s/loosing/losing throughout

Reply:
done